sig-contributor-experience/annual-report-2024.md

```diff
@@ -52,16 +52,17 @@
 
 ## [Working groups](https://git.k8s.io/community/sig-contributor-experience#working-groups)
 
+None
 
 ## Operational
 
 Operational tasks in [sig-governance.md]:
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
-- [ ] Other contributing docs (e.g. in devel dir or contributor guide) reviewed for accuracy and updated if needed
+- [x] [README.md] reviewed for accuracy and updated if needed
+- [x] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
+- [x] Other contributing docs (e.g. in devel dir or contributor guide) reviewed for accuracy and updated if needed
 - [ ] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
```

Comment:
In process of auditing all the OWNERS files.

Reply:
@Priyankasaggu11929 and I have audited the OWNERS files and will update our findings as an issue.